task/sink.py

```diff
@@ -85,10 +85,14 @@ def flush(self, status=None):
         shutil.rmtree(self.attachments)
 
         # All done sending output
-        self.ssh.stdin.close()
+        try:
+            self.ssh.stdin.close()
+        except BrokenPipeError:
```

Comment:
Should I print something here to stderr?

Reply:
I think it is fine I think to not to bother.